src/things_mcp/cache.py

```
@@ -222,19 +223,44 @@ def clear_cache() -> None:
-    
-    thread = threading.Thread(target=cleanup_task, daemon=True)
-    thread.start()
+
+    _cleanup_thread = threading.Thread(target=cleanup_task, daemon=True, name="cache-cleanup")
+    _cleanup_thread.start()
     logger.info(f"Started cache cleanup task with {interval}s interval")
 
+
+def stop_cache_cleanup_task() -> None:
+    """Signal the cache cleanup thread to stop and log confirmation."""
+    global _cleanup_stop_event
```

Ruff (0.15.6)

[warning] Using global for `_cleanup_stop_event` but no assignment is done

(PLW0602)